lib/src/v2/schema.dart

```diff
@@ -6,20 +6,20 @@ import 'package:open_api/src/v2/property.dart';
 class APISchemaObject extends APIProperty {
   APISchemaObject();
 
-  String title;
-  String description;
-  String example;
-  List<String> required = [];
+  String? title;
+  String? description;
+  String? example;
+  List<String?>? isRequired = [];
```

Comment:
since this is not a boolean, it's a bit weird to change this to `isRequired`, should probably remain `required`?

Reply:
Thought it was weirder since `required` is now a reserve word.

Reply:
We could change the name to requirements, but I would rather not keep `required`.

Reply:
i've changed it to `required` in my own fork to keep the existing code working, since I saw no reason to make it more awkward..  but i guess this project is dead anyway 😅️ just thought i might mention it, since it looked like some bulk-fix to me.
fwiw, `required` is not a [reserved word](https://dart.dev/guides/language/language-tour#keywords) and it seems you can even have a parameter list like `void foo({required int required})`

Reply:
Aaa I am mistaken, it's still just a marker but without the @. It still felt weird to me, plus with some linters highlighting the word, it can be distracting to some developers. Either way I think it would still be a better idea to use another word for the variables. We need to discuss who will take ownership of this project in the future and how it fits in with the other aqueduct dependencies.

On a related note, a group of us are going to discuss transitioning aqueduct and the dependencies to an open source community. The conversation is happening right now on Slack, but I would like something more coordinated.